app/src/main/java/com/gruppo3/wetravel/mapManager/types/DestinationMarker.java

```diff
@@ -0,0 +1,81 @@
+package com.gruppo3.wetravel.mapManager.types;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.google.android.gms.maps.model.LatLng;
+
+/**
+ * DestinationMarker object containing the information on a map-displayable marker.
+ *
+ * @author Giovanni Barca
+ */
+public class DestinationMarker {
+    private LatLng latLng = null;
+    private String title = null;
+    private float color;
+    private Object object = null;
+
+    /**
+     * @param latLng Coordinates where to place the marker
+     * @param title String to show when user clicks on the marker (it's showed in a popup over the marker)
+     */
+    public DestinationMarker(@NonNull LatLng latLng, @NonNull String title) {
+        this.latLng = latLng;
+        this.title = title;
+    }
+
+    /**
+     * @param latLng Coordinates where to place the marker
+     * @param title String to show when user clicks on the marker (it's showed in a popup over the marker)
+     * @param color Color of the marker. Should be a float value between 0 and 360, representing points on a color wheel, or can be a BitmapDescriptorFactory predefined color
+     */
+    public DestinationMarker(@NonNull LatLng latLng, @NonNull String title, float color) {
```

Comment:
How about writing in the specifications that these parameters can't be null?
```suggestion
     * @param latLng Coordinates where to place the marker. Never null.
     * @param title String to show when user clicks on the marker (it's showed in a popup over the marker). Never null.
     * @param color Color of the marker. Should be a float value between 0 and 360, representing points on a color wheel, or can be a BitmapDescriptorFactory predefined color
     */
    public DestinationMarker(@NonNull LatLng latLng, @NonNull String title, float color) {
```